include/roboptim/core/detail/utility.hh

```diff
@@ -334,6 +334,17 @@ namespace roboptim
       }
     };
   } // end of namespace detail.
+
+  //ahould probably move this code in another file
+  static bool is_malloc_allowed = true;
```

Comment:
This moves the initial problem to this variable without solving it. We need to move this to `libroboptim-core.so`.